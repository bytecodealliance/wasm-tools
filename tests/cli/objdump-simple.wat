;; RUN: objdump %

(module
  (func)

  (@custom "x" "....")
)
