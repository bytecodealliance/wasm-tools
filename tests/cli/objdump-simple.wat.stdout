  types                                  |        0xa -        0xe |         4 bytes | 1 count
  functions                              |       0x10 -       0x12 |         2 bytes | 1 count
  code                                   |       0x14 -       0x18 |         4 bytes | 1 count
  custom "x"                             |       0x1c -       0x20 |         4 bytes | 1 count
